assets/js/renderer.js

```diff
@@ -355,8 +355,51 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Clear any saved splitter-related div widths from local storage when the app closes.
+    const clearDivs = () => {
```

Comment:
Typically, instead of assigning a fat arrow to a const variable, just use a normal function, like:

```
function clearDivs() {
    ...
}
```